plugins/glustershd/actor.go

```diff
@@ -32,7 +32,7 @@ func (actor *shdActor) Do(v *volume.Volinfo, key string, value string, logger lo
 	case "on":
 		if isHealEnabled(v) {
 			err = daemon.Start(glustershDaemon, true, logger)
-			if err != gderrors.ErrProcessAlreadyRunning || err != nil {
+			if err != gderrors.ErrProcessAlreadyRunning && err != nil {
```

Comment:
While this okay, for slightly better readablity, can you please check for nil first and then the specific error ?

Below too

Reply:
sure